pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ClientCachedDao.java

```diff
@@ -0,0 +1,119 @@
+package org.broadinstitute.ddp.db.dao;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+import java.io.Serializable;
+import java.util.Collection;
+import java.util.List;
+
+import javax.cache.Cache;
+import javax.cache.expiry.Duration;
+
+import org.broadinstitute.ddp.cache.CacheService;
+import org.broadinstitute.ddp.db.dto.ClientDto;
+import org.broadinstitute.ddp.security.StudyClientConfiguration;
+import org.jdbi.v3.core.Handle;
+
+public class ClientCachedDao extends SQLObjectWrapper<ClientDao> implements ClientDao {
```

Comment:
Don't see the `SqlObjectWrapper` file yet. Can you push up that commit?

Reply:
Oops! Just pushed it.